cobalt/cobalt_main_delegate.cc

```diff
@@ -37,6 +38,7 @@ CobaltMainDelegate::~CobaltMainDelegate() {}
 absl::optional<int> CobaltMainDelegate::BasicStartupComplete() {
   base::CommandLine* cl = base::CommandLine::ForCurrentProcess();
   cl->AppendSwitch(switches::kEnableAggressiveDOMStorageFlushing);
+  cl->AppendSwitch(switches::kDisableGpuShaderDiskCache);
```

Comment:
Tiny nitpick, but i almost would have submitted this as a separate PR - easier to reason about and find.